app/src/androidTest/java/com/github/se/cyrcle/ui/review/ReviewScreenTest.kt

```diff
@@ -75,7 +75,7 @@ class ReviewScreenTest {
     composeTestRule
         .onNodeWithTag("TopAppBarTitle")
         .assertIsDisplayed()
-        .assertTextContains("Add Your Review")
+        .assertTextContains("Add your review")
```

Comment:
Why remove the captialization? It's a title...

Reply:
It's true. I just tried removing some inconsistent caps in the strings ressources without really overthinking it. Do you really think it's ok to have "Add Your Review"? To me it looks illegal. Anyways we can come back to these details later, let's keep moving forward witht the important stuff!

Reply:
Completely agree!